vertx-web/src/main/asciidoc/java/index.adoc

```diff
@@ -47,10 +47,11 @@ Some of the key features of Vert.x-Web include:
 * Favicon handling
```

Comment:
You should not edit the asciidoc files manually because they are generated from:

https://github.com/vert-x3/vertx-web/blob/master/vertx-web/src/main/java/io/vertx/ext/web/package-info.java#L1332

Please apply the changes there and during the build process the asciidocs are updated to all languages. You can even add examples that if the code is simple enough will be translated to all supported languages. Examples must be written in Java.